conf/modules/rank_variants.config

```diff
@@ -0,0 +1,64 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
+        ext.when            = Conditional clause
+----------------------------------------------------------------------------------------
+*/
+
+//
+// Score and rank SNVs
+//
+
+process {
+
+    withName: '.*:RANK_VARIANTS_SNV:GENMOD_ANNOTATE' {
+        ext.prefix = { "${meta.id}_snv_genmod_annotate" }
+        ext.args = { [
+            '--annotate_regions',
+            '--genome-build 38',
```

Comment:
Will the pipeline be available only for 38? Otherwise you could do something like `params.genome`